jsonexamples/generated/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(generated_files
```

Comment:
I know this works, obviously, but I find it confusing. Nothing else in the project seems to depend on generated_files. But  benchmark/benchfeatures.cpp requires these files... Gosh. How does this work? And what happens when ruby is missing... how can benchfeatures work?

Reply:
When ruby is missing, benchfeatures will fail :) This is all manual right now: you have to `make benchfeatures generated-data` and run it yourself. Ultimately, I'd like to write the generator in something other than Ruby (probably C++, just because that's what we have available) and make it a full target that builds and runs.

Reply:
> When ruby is missing, benchfeatures will fail :)

This is not a request on my part... but wouldn't the nice thing be to make it so that the benchfeatures is disabled if ruby is missing? (I am not asking for changes, just inquiring.)

Reply:
I thought about that, but I kinda want to keep it compiling in CI if we can. I'd rather it not rot again.